package.json

```diff
@@ -75,6 +75,7 @@
     "jest": "^28.1.0",
     "jest-environment-jsdom": "^28.1.0",
     "lodash-es": "^4.17.21",
+    "material-ui-popup-state": "^5.0.4",
```

Comment:
What's your workflow for adding dependencies? I typically would run `yarn add material-ui-popup-state`.  It doesn't look like this library requires anything that we don't already have, so all these tarballs shouldn't be showing up.

Reply:
Yeah, that's what I did. I don't know why it would include those if we already had the deps